### PR TITLE
Set top ranked pharmacy search radius to 15mi

### PR DESCRIPTION
**Pull request by jknapp25**
Right now we are returning costcos that are real far away (see below) for top ranked pharmacies like Costco we should limit the radius in search.

image (40)

### CODE REVIEW

File: apps/patient/src/api/internal.ts [hunk 1]
```diff
@@ -53,7 +54,7 @@ export const getPharmacies = async ({
       GET_PHARMACIES,
       {
         location: {
-          radius: 100,
+          radius: searchParams.radius ?? 100,
```
Comment (marisbest2) [inline apps/patient/src/api/internal.ts:57]: These look like they're the same?

File: apps/patient/src/views/Pharmacy.tsx [hunk 2]
```diff
@@ -245,7 +245,7 @@ export const Pharmacy = () => {
       }
       try {
         const topRankedCostco: EnrichedPharmacy[] = await getPharmacies({
-          searchParams: { latitude, longitude },
+          searchParams: { latitude, longitude, radius: 15 },
```
Comment (marisbest2) [inline apps/patient/src/views/Pharmacy.tsx:248]: This is what matters

File: apps/patient/src/api/internal.ts (hunk 1 above)
Comment (jknapp25) [reply apps/patient/src/api/internal.ts:57]: not sure i understand

Comment (jknapp25) [reply apps/patient/src/api/internal.ts]: got it

**Review by marisbest2 — APPROVED** [review]

**Pull request merged**